Mathlib/Data/Set/Lattice.lean

```diff
@@ -2081,8 +2081,7 @@ section Seq
 
 /-- Given a set `s` of functions `α → β` and `t : Set α`, `seq s t` is the union of `f '' t` over
 all `f ∈ s`. -/
-def seq (s : Set (α → β)) (t : Set α) : Set β :=
-  { b | ∃ f ∈ s, ∃ a ∈ t, (f : α → β) a = b }
+def seq (s : Set (α → β)) (t : Set α) : Set β := image2 (fun f a ↦ f a) s t
```

Comment:
Do you think this is better?
```suggestion
def seq (s : Set (α → β)) (t : Set α) : Set β := image2 id s t
```

Reply:
`Function.eval` would probably be clearest.

Reply:
Yep, that's backward.

Reply:
I reverted this commit because it generates `id` here and there after `simp only [seq]`.

Reply:
What about
```suggestion
def seq (s : Set (α → β)) (t : Set α) : Set β := image2 (fun f ↦ f) s t
```
then?